src/assets/javascript/application.js

```diff
@@ -16,7 +16,9 @@ function initChangePhoneNumber() {
     intPhoneNumberCheckbox.addEventListener("click", function(event) {
       onIntNumberSelected(event.currentTarget, phoneNumberInput);
     });
-    window.addEventListener("load", onIntNumberSelected(intPhoneNumberCheckbox, phoneNumberInput));
+    window.addEventListener("load", function(event) {
```

Comment:
This change should be in separate PR for (ATB-206)[https://govukverify.atlassian.net/browse/ATB-206]

Reply:
Separated :)